.travis.yml

```diff
@@ -50,7 +50,7 @@ script:
       travis_terminate 1
       ;;
     esac'
-  - pip install -r requirements.txt
```

Comment:
The problem is that the  requirement depend on the installed version of elasticsearch. We have to install the proper version of elasticsearch_dsl. Is there a way to doing that properly in requirements.txt ?